src/app/routes/liveRadio/getInitialData/index.js

```diff
@@ -44,6 +44,13 @@ export default async ({ path: pathname, pageType, service, toggles }) => {
 
     const getRadioScheduleData = path(['radioScheduleData']);
 
+    const mediaLoaderBlock = [
```

Comment:
For consistency with our other page types (work currently in progress: #11928), can we rename this to `mediaBlock`?